people/lucia-specia.md

```diff
@@ -0,0 +1,30 @@
+---
+description: Researcher and professor of natural language processing
+---
+# Lucia Specia
+Lucia Specia is a researcher and professor of natural language processing. 
+In 2007, Specia received Ph.D. in computer science from the University of São Paulo. 
+
+Her main research interests and activities focus on:
+* machine translation
+* image captioning
+* quality estimation
+* text adaptation
+
+Specia led early research on machine translation [quality estimation](/quality/quality-estimation.md) (QE). Her contribution to the matter includes: 
```

Comment:
I think this sentence should go right at the top (i.e. be the second line in the article).

Then here we can just say:

> Her work on quality estimation includes: